client/src/context/ProfileContext.js

```diff
@@ -64,14 +68,31 @@ const ProfileState = (props) =>{
 			});
 	};
 
+	const getSkills = ()=>{
+		dispatch({ type: types.Set_Is_Loading }),
+		axios.get(`${baseUrl}/graduates`)
+			.then((response)=>{
+				dispatch({ type: types.Set_All_Profiles, payload:response.data });
+			})
+			.catch((error)=>{
+				dispatch({ type:types.Set_Error, payload:error });
+			});
+	};
+
+
+	// .then((response)=>{
+	// 	// 	const graduate = response.data[0];
+	// 	// 	console.log('graduate', graduate),
+	// 	// 	dispatch({ type:types.Set_Profile, payload: graduate });
+	// 	// })
 
 	const getProfile =  (id) => {
-		console.log('get profile called')
 		dispatch({ type: types.Set_Is_Loading });
-		axios.get(`https://gist.githubusercontent.com/OBakir90/46c0de835cb3db4c42f655e5f467825a/raw/d16c488a33cc1ebbceea866fe988591c3683bf0c/myprofile.json`)
+		axios.get(`${baseUrl}/graduates/${id}`)
 			.then((response)=>{
-				console.log('response', response);
-				dispatch({ type: types.Set_Profile, payload:response.data[0] });
+				const graduate = response.data[0];
+				graduate.skills=response.data.map(x=>x.skill_name)
```

Comment:
Could you please show me an example of `response.data` here? Not sure I understand why you need to map through the whole array when you're taking the first element of that same array as the graduate?